app/controllers/IdentityController.scala

```diff
@@ -41,12 +41,13 @@ class IdentityController(
       .setPasswordGuest(request.body.password, request.body.guestAccountRegistrationToken)
       .fold(
         err => {
-          SafeLogger.error(scrub"Failed to set password")
+          SafeLogger.error(scrub"Failed to set password: ${err.toString}")
           InternalServerError(err.asJson)
```

Comment:
does the err go back to the client side?  probably better to be circumspect about what went wrong (I know this isn't your change though)

Reply:
It does currently. What exactly do you mean by `be circumspect about what went wrong`? what do you think it should return? Just the InternalServerError without the `err.json`?

Reply:
yes just the internal server error maybe saying "something went wrong, please try again later" or similar
Here we are not controlling what goes into the response, for example in a worst case it could contain a toString of a case class that contains an identity api key, or other information useful to an attacker.
There's no need to include the information, because we only should return something that can help the user take action, but if it's only some internal error there's not much they can do in response.

Reply:
👍